eas.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Do we want/need this to be in version control? I guess we can add it for now and if it ever holds secrets down the line, we could add it to the git ignore.